src/map/utils/battleutils.cpp

```diff
@@ -5205,30 +5229,7 @@ namespace battleutils
                 }
             }
         }
-        else if (PSpell->getSpellGroup() == SPELLGROUP_SONG)
-        {
-            if (PEntity->StatusEffectContainer->HasStatusEffect(EFFECT_NIGHTINGALE))
-            {
-                recast *= 0.5f;
-            }
-            // The following modifiers are not multiplicative - as such they must be applied last.
-            // ShowDebug("Recast before reduction: %u\n", recast);
-            if (PSpell->getID() == 462) // apply Finale recast merits
-            {
-                recast -= ((CCharEntity*)PEntity)->PMeritPoints->GetMeritValue(MERIT_FINALE_RECAST, (CCharEntity*)PEntity) * 1000;
-            }
-            if (PSpell->getID() == 376 || PSpell->getID() == 377 || PSpell->getID() == 463 || PSpell->getID() == 471) // apply Lullaby recast merits
-            {
-                recast -= ((CCharEntity*)PEntity)->PMeritPoints->GetMeritValue(MERIT_LULLABY_RECAST, (CCharEntity*)PEntity) * 1000;
-            }
-            recast -= PEntity->getMod(MOD_SONG_RECAST_DELAY);
-            // ShowDebug("Recast after merit reduction: %u\n", recast);
-        }
 
-        if (recast < 0)
```

Comment:
I put that in in my previous commit when I added the subtraction mods for BRD songs. Since they're done before the recast gets set to a minimum of 20% of base, if it were to be negative it'd still get set to 20% of base.